web/src/elements/forms/Radio.ts

```diff
@@ -15,7 +17,7 @@ export interface RadioOption<T> {
 }
 
 @customElement("ak-radio")
```

Comment:
Provide a more consistent interface to the RadioButton. First, we dispatch the  events to the outside world, and we set the value locally so that the current `Form.ts` continues  to behave as expected. We also prevent the "button lost value" event from propagating; this  presents a unified select-like interface to users of the RadioButtonGroup. The current value  semantics are preserved; other clients of the RadioButton do not see a change in behavior.